local_planner/src/nodes/common.h

```diff
@@ -31,8 +31,11 @@ bool hasSameYawAndAltitude(geometry_msgs::PoseStamped old_wp,
                            double old_yaw);
 double elevationIndexToAngle(int e, double res);
 double azimuthIndexToAngle(int z, double res);
-int azimuthAnglefromCartesian(double x, double y, double z,
-                              geometry_msgs::Point pos);
+int azimuthAnglefromCartesian(geometry_msgs::Point position,
```

Comment:
these could be `const geometry_msgs::Point &position`

Reply:
I've done this now for all of the larger types in common